UnityProject/Assets/Scripts/Items/Medical/DefibrillatorPaddles.cs

```diff
@@ -34,13 +34,17 @@ void Perform()
 			var LHMB = interaction.TargetObject.GetComponent<LivingHealthMasterBase>();
 			foreach (var BodyPart in LHMB.ImplantList)
 			{
-				var heart = BodyPart as Heart;
-				if (heart != null)
+				foreach (var bodyPartModification in BodyPart.BodyPartModifications)
 				{
-					heart.HeartAttack = false;
-					heart.CanTriggerHeartAttack = false;
-					heart.CurrentPulse = 0;
+					var heart = bodyPartModification as Heart;
```

Comment:
same here, use ``is Heart`` instead and remember to break the for in case it found a heart or do we want to have multiple hearts?

Reply:
multiple hearts is good